src/main/kotlin/com/team4099/robot2022/subsystems/Drivetrain.kt

```diff
@@ -335,7 +338,7 @@ object Drivetrain : SubsystemBase() {
   }
 
   fun zeroGyro() {
-    // gyro.reset() TODO
+    gyro.angleAdjustment = gyroChange.inDegrees
```

Comment:
can you not simplify this by getting rid of `gyroChange` and just do `gyroOffset = -gyroAngle` here?

also you can combine this with the below method using kotlin optional parameters (one method where `offset: Angle = 0.degrees` like you would in python)

Reply:
or in other words gyro offset can be private and it doesn't matter if its storing the offset you provided it or the offset calculated for zeroing or some combination of both

Reply:
also for the parameter, not the field, i would name it `toAngle` or something so its clear which way its offsetting (like to make it clear that the offset is not being subtracted or something, and instead we're saying that we're "zeroing" to the given angle